spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -391,6 +390,15 @@ to facilitate the apps binding to the RabbitMQ service. Assuming your RabbitMQ s
 ```
 cf set-env dataflow-server SPRING_APPLICATION_JSON '{"spring.cloud.dataflow.applicationProperties.stream.spring.rabbitmq.addresses": "${vcap.services.rabbit.credentials.protocols.amqp.uris}"}'
 ```
+For Spring Cloud Task apps, something similar to the following could be used, if using a database service instance named `mysql`:
+
+```
+cf set-env dataflow-server spring.cloud.dataflow.applicationProperties.task.spring.datasource.url '${vcap.services.mysql.credentials.jdbcUrl}'
```

Comment:
Sorry, could you clarify (maybe putting your manifest.yml here).
If you set `SPRING_DATASOURCE_URL` as an env var to the server, then it is just that (sets that value, which boot can understand, *for the server*). Those are *not* propagated to deployed apps by default. I must be missing something here